src/extension.ts

```diff
@@ -160,6 +161,7 @@ export function activate(context: vscode.ExtensionContext): void {
             void vscode.env.openExternal(vscode.Uri.parse('https://github.com/microsoft/vscode-edge-devtools/blob/master/README.md'));
         }));
     void vscode.commands.executeCommand('setContext', 'titleCommandsRegistered', true);
+    void reportFileExtensionTypes(telemetryReporter);
```

Comment:
Is everyone okay with checking the extension types upon activation?  I didn't want it to fire on launch just so we don't dilute this event.  Open to suggestions

Reply:
Reporting on activation sounds good to me. I'm not 100% sure but I think if the user opens a new folder, the window will reload and activate the extension again. So if that's the case, we should get reports from multiple projects.
Also, I did a little digging to the textDocuments property of the vscode.workspace earlier, and this seems to be the files that are open in the editor at a given time. It could potentially be useful to see which files they actually have open when they launch the extension (as opposed to the files that make up the entire project), but that should be a separate event I think.

Reply:
Double checked and opening a new folder will re-launch VS Code and will refire the telemetry event.  However, opening a file will not relaunch nor refire, but I don't think we care about that scenario too much anyway.